cime_config/machines/config_batch.xml

```diff
@@ -648,12 +648,58 @@
      </queues>
    </batch_system>
 
+   <batch_system MACH="spock" type="slurm">
```

Comment:
I think this xml tag also needs to be removed from SCREAM, as it is a duplicate of the spock tag that appears earlier in this file.

Reply:
@oksanaguba are you able to take this cleanup task?

Reply:
https://github.com/E3SM-Project/scream/pull/2179 will remove the duplicate tags. I will reset this branch once that PR is merged into scream master.